dpsim/src/MNASolverDirect.cpp

```diff
@@ -44,14 +43,18 @@ void MnaSolverDirect<VarType>::switchedMatrixStamp(std::size_t index, std::vecto
 
 	// Compute LU-factorization for system matrix
 	mDirectLinearSolvers[bit][0]->preprocessing(sys, mListVariableSystemMatrixEntries);
-
+	auto start = std::chrono::steady_clock::now();
 	mDirectLinearSolvers[bit][0]->factorize(sys);
+	auto end = std::chrono::steady_clock::now();
+	std::chrono::duration<Real> diff = end-start;
+	mFactorizeTimes.push_back(diff.count());
 }
 
 template <typename VarType>
 void MnaSolverDirect<VarType>::stampVariableSystemMatrix() {
 
-	this->mDirectLinearSolverVariableSystemMatrix = createDirectSolverImplementation();
+	this->mDirectLinearSolverVariableSystemMatrix = createDirectSolverImplementation(mSLog);
+	this->mDirectLinearSolverVariableSystemMatrix->setConfiguration(configurationInUse);
```

Comment:
Add TODO comment that - so far - a direct linear solver configuration is applied only in case of system matrix recomputation

Reply:
added